examples/hosts/app-integration/external-data/package.json

```diff
@@ -33,11 +33,13 @@
     "prettier:fix": "prettier --write . --ignore-path ../../../../.prettierignore",
     "rebuild": "npm run clean && npm run build",
     "start": "start-server-and-test start:tinylicious 7070 start:app",
-    "start:app": "start-server-and-test start:mock-service 5237 start:client",
-    "start:app:test": "start-server-and-test start:mock-service 5237 start:client:test",
+    "start:app": "start-server-and-test start:services \"5236|5237\" start:client",
+    "start:app:test": "start-server-and-test start:services \"5236|5237\" start:client:test",
     "start:client": "webpack serve",
     "start:client:test": "webpack serve --config webpack.test.js",
-    "start:mock-service": "cross-env MOCK_CUSTOMER_SERVICE_PORT=5237 node dist/src/mock-service/start.js",
+    "start:customer-service": "cross-env CUSTOMER_SERVICE_PORT=5237 node dist/src/mock-customer-service/start.js",
+    "start:external-data-service": "cross-env EXTERNAL_DATA_SERVICE_PORT=5236 node dist/src/mock-external-data-service/start.js",
+    "start:services": "start-server-and-test start:external-data-service 5236 start:customer-service",
```

Comment:
Note: the order here is important. The customer service registers for webhook notifications from the external data service upon setup, so the external data service has to be launched first.